### DISCUSSION TITLE
ok so the nuke is primed, armed, and ready to drop.

### DISCUSSION THREAD

**Discussion by rwg100** [3kh0/ext-remover#292] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
so im not sure where i should drop, but im in a b-29 "superfortress" bomber. not the enola gay. i have a 847 kiloton nuke with plutonium, i think ill land it in georgia, somewhere 2 hours away from atlanta.

currently 347 knots, altitude it 2389 ft above ground.

dropping in 3 hours

**Comment by 3kh0** [comment, marked as answer]
ohio

**Comment by Aka-but-nice** [comment]
True, in Ohio it's a literal battle ground.
